test/performance/dataplane-probe/sla.go

```diff
@@ -41,6 +41,12 @@ var (
 				ValueKey:            proto.String("kd"),
 			},
 		}},
+		CrossRunConfig: &tpb.CrossRunConfig{
```

Comment:
too many duplicates. Can we have a common var and use that? Maybe even move it to mako package so that all tests can use this default?

Reply:
Moved to using a separate function to construct the config. Added a placeholder for `tags` in case we want to search for historic runs that has the same tags.